gender_novels/common.py

```diff
@@ -104,7 +104,7 @@ def load_file_locally(file_path, current_file_type):
         # repo so it returns the correct path. But it will change once
         # this function gets moved.
         local_base_path = Path(os.path.abspath(os.path.dirname(__file__)))
-        file = open(local_base_path.joinpath(file_path), mode='r')
+        file = open(local_base_path.joinpath(file_path), mode='r', encoding='utf8')
```

Comment:
Are you sure this will do anything?  I think we want io.open with errors='ignore' or something like that.